db/data/20210319102418_make_organisations_consistent.rb

```diff
@@ -0,0 +1,23 @@
+# Run me with `rails runner db/data/20210319102418_make_organisations_consistent.rb`
+
+# Put your Ruby code here
+
+service_owner = Organisation.find_by(service_owner: true)
+
+def extending_organisation_for_activity(activity)
+  if activity.level == "programme"
+    activity.children ? activity.children.first.organisation : activity.organisation
+  else
+    activity.parent.organisation
```

Comment:
I'm getting mired in organisations and extending organisations, parents and children, so I might have it wrong here, but:

For a *project* activity, the extending organisation should be the delivery partner org, if we know it. If the ext org for a project is nil, we have to get the info elsewhere.

But if we get the organisation of the parent activity, which might be (must be?) a *programme*, the org of the programme is going to be BEIS, the `service_owner`, so we end up with the extending org of the project being BEIS, when it should be the delivery partner (*if* we have this info somewhere in the programme).

Is this what is intended by this code?

Reply:
We've removed this commit from the PR, because the data migration wasn't fixing all places where the data was inconsistent.

I've also realised that the activity policy files should get updated at the same time, as they make assumptions based the way `extending_organisation` was used quite a while ago.